relops_hardware_controller/celery.py

```diff
@@ -3,13 +3,25 @@
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 
 import os
+import logging
+import json
+try:
+    from StringIO import StringIO
```

Comment:
ugh Python 2 sadness

Reply:
since this is on python3 in the container, I'll clean this out